core/src/avm2/object/loaderinfo_object.rs

```diff
@@ -264,7 +264,7 @@ impl<'gc> LoaderInfoObject<'gc> {
         context: &mut UpdateContext<'gc>,
         status: u16,
         redirected: bool,
-    ) {
+    ) -> bool {
```

Comment:
can you add a comment on the return value?
Without some documentation, it took me a moment to understand what the `return true; } return false; } true` chain corresponds to